Coding-Coach/coding-coach
# Update Input Style

# Details

## Description

This adds styles for focused and errored text inputs. In the case of the error styling, an `error<string>` prop will need to be passed into the `TextField` component.

## Steps to QA

1. Run `yarn run storybook`
2. Go to Components > TextInput
3. Check out the stories.

## Issue

Closes #230
